cpp/benchmarks/string/substring.cpp

```diff
@@ -23,8 +23,8 @@
 #include <cudf_test/column_wrapper.hpp>
 
 #include <cudf/scalar/scalar.hpp>
+#include <cudf/strings/slice.hpp>
```

Comment:
For naming consistency, shall we rename this file as well as the benchmarks?

Reply:
Yes, I think that makes sense.